src/test/java/com/rakovets/course/java/core/practice/lambda_expressions/TaskOneTest.java

```diff
@@ -0,0 +1,26 @@
+package com.rakovets.course.java.core.practice.lambda_expressions;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskOneTest {
+    @Test
+    void mapTest() {
+        String[] array = {"-i", "in.txt", "--limit", "40", "-d", "1", "-o", "out.txt"};
+        String actual = new TaskOne().getMap(array).toString();
+        assertEquals("{-o=out.txt, -d=1, -i=in.txt, --limit=40}", actual);
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"